### ISSUE TITLE
Incoming call not displaying it's a video call

### ISSUE THREAD

**Issue by ThomasLarge** [react-native-webrtc/react-native-callkeep#54]
I have a function that is being called when I get a notification but I can't get it to say Video and not audio. Bellow is the function

```
onIncomingCall(callUuid) {
   RNCallKeep.displayIncomingCall(callUuid, "Thomas Large", null, true);
}
```

True is the way the docs tells you how to set it

Can anyone help?

**Comment by tienviip** [comment]
hi @ThomasLarge have you found a solution for videocall, please support me.

**Comment by ThomasLarge** [comment]
@tienviip do you mean how to get it working? or the issue I have above?

**Comment by tienviip** [comment]
@ThomasLarge  yep, I also encountered your problem above, Can you help me, please.

**Comment by ThomasLarge** [comment]
@tienviip I'm waiting on someone to help me too 😆 I'm going to try and dig into it later but haven't got the time at the moment that's why I have opened the issue

**Comment by LinusU** [comment]
The problem is probably that `hasVideo` is hard-coded to `NO` here:

https://github.com/react-native-webrtc/react-native-callkeep/blob/7d7306027834da68f488b4d88bb1ad9910e9348b/ios/RNCallKeep/RNCallKeep.m#L135

**Comment by ThomasLarge** [comment]
@LinusU Looks like it's here too 

https://github.com/react-native-webrtc/react-native-callkeep/blob/7d7306027834da68f488b4d88bb1ad9910e9348b/ios/RNCallKeep/RNCallKeep.m#L254

Do you think it's best to change the file locally or is it best to wait until the repo has been updated?

**Comment by danjenkins** [comment]
Update it locally, see if it fixes your issue and then submit a PR with it
configurable :) in all likelihood it won't get fixed unless you fix it!

**Comment by ThomasLarge** [comment]
@danjenkins & @LinusU I have changed that file and it doesn't seem to work still I think there's more places it's hardcoded or it's something that isn't working in the package. I haven't got the time over the next couple of days to look into it more but I will over the weekend. There's also these two lines 

https://github.com/react-native-webrtc/react-native-callkeep/blob/5cae45787f46fb772a31369c8bc89fd5167c2033/index.js#L48

https://github.com/react-native-webrtc/react-native-callkeep/blob/5cae45787f46fb772a31369c8bc89fd5167c2033/index.js#L57

**Comment by jaswantjatav** [comment]
HI anyone  found a solution for showing incoming call ui, please help me, I am using react-native-WebRTC  plugin , react-native-callkeep and want to show incoming call ui but nothing showing even I called the function
RNCallKeep.displayIncomingCall("978697869", "TESting");

But no luck, please help me if anyone got solution.

**Comment by danjenkins** [comment]
@jaswantjatav if you need help then please make a new issue

**Comment by danjenkins** [comment]
@ThomasLarge @LinusU 

So you need to change https://github.com/react-native-webrtc/react-native-callkeep/blob/7d7306027834da68f488b4d88bb1ad9910e9348b/ios/RNCallKeep/RNCallKeep.m#L135 's `NO `to `hasVideo`

I'd just comment out the line https://github.com/react-native-webrtc/react-native-callkeep/blob/7d7306027834da68f488b4d88bb1ad9910e9348b/ios/RNCallKeep/RNCallKeep.m#L254 

@ThomasLarge those 2 links to the index.js are just setting a default of false. If you pass in true it'll be true :)

**Comment by ThomasLarge** [comment]
Cheers @danjenkins I will look at this tomorrow or the weekend

**Comment by ThomasLarge** [comment]
I can confirm this is all fixed 👍
